pkg/test/framework/components/echo/check/checkers.go

```diff
@@ -249,6 +250,16 @@ func Hostname(expected string) echo.Checker {
 	})
 }
 
+// Hostnames check whether the hostname the request landed on is from the expected slice. This differs from Host which is the request we called.
+func Hostnames(expects []string) echo.Checker {
```

Comment:
nit: I think the naming needs to be more distinct from Host. Maybe like `ResponseHost`?